packages/manager/modules/pci/src/projects/project/edit/edit.html

```diff
@@ -26,23 +27,48 @@
             required
         />
     </oui-field>
+    <oui-field
+        label="{{ ::'pci_projects_project_edit_project_name' | translate }}"
+        label-popover="{{ ::'pci_projects_project_edit_discovery_disable_tooltip' | translate }}"
+        data-ng-if="$ctrl.isDiscoveryProject"
+        size="xl"
+    >
+        <input
+            class="oui-input"
+            type="text"
+            id="description"
+            name="description"
+            data-ng-disabled="$ctrl.isDiscoveryProject"
```

Comment:
**nitpick**: If the element is displayed, we don't need a second check
Also, maybe we can just use readonly instead of disabling the input
```suggestion
            readonly
```